docs/developer-tools/ts-sdk-intro.md

```diff
@@ -4,24 +4,43 @@ sidebar_position: 2
+5. **Integration with the Runtime** : We aim to provide a Runtime Rest Client which is 1-1 feature parity with [`marlowe-runtime-web`](https://github.com/input-output-hk/marlowe-cardano/tree/main/marlowe-runtime-web), so you can benefit from all the Runtime features in a JavaScript and TypeScript Environment (see [@marlowe.io/runtime-rest-client](https://input-output-hk.github.io/marlowe-ts-sdk/modules/_marlowe_io_runtime_rest_client.html)).
+6. **Coordination Between Wallets and Runtime** : We provide abstractions over the Runtime Rest Client and the Wallets that remove "plumbing" logic and allows you to focus on your core business logic (see [@marlowe.io/runtime-lifecycle](https://input-output-hk.github.io/marlowe-ts-sdk/modules/_marlowe_io_runtime_lifecycle.html)).    
+7. **Prototype DApp examples**: Embark on your DApp journey using the TS-SDK's distilled prototype examples. These prototypes serve as launching pads for your customized applications.
 
 ## Getting started
-To use the capabilities of the Marlowe TS-SDK, you will need to have a URL to a running instance of the Marlowe Runtime and a compatible wallet extension installed in your browser. Please refer to the **[Marlowe starter kit](https://github.com/input-output-hk/marlowe-starter-kit)** for guidance about accessing a running instance of the Runtime. 
 
-## Prototype examples
+Please look at the [README.md](https://github.com/input-output-hk/marlowe-ts-sdk/) of the Github Repository.. Examples, Detailed Documentation, small tools are available, allowing a quick overview of the Marlowe TS-SDK and its capabilities. 
+
+This **[Marlowe TypeScript SDK (TS-SDK)](https://github.com/input-output-hk/marlowe-ts-sdk/)** is released under an open source license, it can be read, used, deployed, customized, improved, forked at will. Look at different parts of our Github Repository and especially the [Discussions](https://github.com/input-output-hk/marlowe-ts-sdk/discussions) Section.  
+
+Don't hesitate to provide feedback and particapte to its development, the team will be supportive and will continue to progress in that direction. 
+
+## Marlowe Contract Examples 
+
+These contract examples are released under an open source license within our npm package [`@marlowe.io/language-examples`](https://input-output-hk.github.io/marlowe-ts-sdk/modules/_marlowe_io_language_examples.html).They can be read, customized, improved, forked at will. 
+
+Don't hesitate to provide feedback, the team will be supportive and will continue to progress in that direction.
+
+- **[Vesting]([https://github.com/input-output-hk/marlowe-ts-sdk/blob/main/pocs/contract-example/vesting-flow.html](https://input-output-hk.github.io/marlowe-ts-sdk/modules/_marlowe_io_language_examples.vesting.html))**: A Vesting Contrat implementation is available in the TS-SDK. It supports the Token Plans Prototype and give you a full implementation of Marlowe Contract used in a Typescript Web-based DApp Environement.
```

Comment:
spelling: Contract... Environment

Malformed markdown link - the example contract should be elsewhere